doc/UsersGuide/source/packagemanager.rst

```diff
@@ -1,70 +1,143 @@
+---------------
+
+The Open Source Modelica Consortium (OSMC) maintains a collection of publicly available, open-source Modelica packages
+on its servers. They are routinely tested with past released versions of OpenModelica, as well as with the current development
+version on the master branch, see the `overview report <https://libraries.openmodelica.org/branches/overview-combined.html>`_.
+Based on the testing results and on information gathered from the library developers, these packages are classified
+in terms of level of support in OpenModelica. Backwards-compatibility information is also collected from the
+conversion annotations.
+
+The OpenModelica Package Manager relies on this information to install the best versions of the library dependencies of your 
+own Modelica packages. It can be run both from the OMEdit GUI and from the command-line interactive environment.
+
+Note that the Package Manager may install multiple builds of the same library version on your PC, if they are available on the
+OSMC servers. When this happens, they are distinguished among each other by means of 
+`semver <https://https://semver.org/#semantic-versioning-specification-semver>`_-style pre- or post-release metadata in the
+top directory name on the file system. Post-release builds are denoted by a plus sign (e.g. ``2.0.0+build.02``)
+and have higher priority over the corresponding plain release
+(e.g. ``2.0.0``), while pre-release builds are denoted by a minus sign (e.g. ``2.0.0-dev.30``) and have a lower priority.
+
+When loading a certain version of a library, unless a specific build is explicitly referenced, the one with higher
+precedence will always be loaded. For example, if the versions ``2.0.0-beta.01``, ``2.0.0``, and ``2.0.0+build.01``
+are installed, the latter is loaded by libraries with uses annotation requiring version ``2.0.0``. Unless, of course,
+there are later backwards-compatible versions installed, e.g., ``2.0.1``, in which case the one with the highest release
+number and priority is installed.
+
+Package Management in OMEdit
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+TBD: Show how to install new packages and manage installed ones
+
+Running Conversion Scripts in OMEdit
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+TBD: Show how to run conversion scripts in OMEdit
+
+Automatically Loaded Packages in OMEdit
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+When you start OMEdit, some packages are automatically loaded into the environment, and shown in the Libraries
+Browser. You can configure which ones are loaded from the Tools|Options|Libraries menu.
+
+Please note that automatically loaded libraries may be in conflict with the dependencies of packages that you may
+then load from the File menu. For example, if you automatically load MSL ``4.0.0``, and then open a library that
+uses MSL ``3.2.3``, you get a conflict, because the former is not backwards-compatible with the latter. In this
+case you have three options: 
+
+- cancel the operation;
```

Comment:
There is no `cancel the operation`. The other two options are fine.

Reply:
See #8183, let's continue the discussion there and then we'll update the documentation accordingly